annotations/quadriga.py

```diff
@@ -275,3 +257,150 @@ def parse_response(raw_response):
         tag = child.tag.replace(QDNS, '')
         data[tag] = child.text
     return data
+
+
+
+def build_concept_node(concept, user, creation_time, source_uri):
+    """
+    Helper function to build a concept node dictionary.
+    """
+    # Get all appellations that created this concept and add stringRep (annotated text)
+    appellations = Appellation.objects.filter(interpretation=concept)
+    
+    term_parts = []
+    for appellation in appellations:
+        term_parts.append({
+            "position": appellation.startPos if appellation.startPos else 0,
+            "expression": appellation.stringRep or "",
+            "normalization": "",
+            "formattedPointer": "",
+            "format": ""
+        })
+    
+    return {
+        "label": concept.label or "",
+        "metadata": {
+            "type": "appellation_event", 
+            "interpretation": concept.uri,
+            "termParts": term_parts
```

Comment:
per appellation event, there should only be the term parts belonging to that appellation (not all appellations with the same concept).